### DISCUSSION TITLE
What is that plugin that animates the current code scope and how to toggle the animation off?

### DISCUSSION THREAD

**Discussion by idelice** [LazyVim/LazyVim#1793] [General]
<img width="378" alt="Screenshot 2023-10-18 at 20 22 14" src="https://github.com/LazyVim/LazyVim/assets/59118341/b627e3de-823f-44c5-8e91-62dd5073903e">

**Comment by dpetka2001** [comment]
`indent-blankline`

**Comment by folke** [comment]
That's. Mini.indentscope.

**Comment by planetsLightningArrester** [comment]
For anyone searching for an exact answer on how to disable ONLY de animation (keep de highlight)

```lua
require("mini.indentscope").setup({
  symbol = "│",
  options = { try_as_border = true },
  draw = {
    animation = require("mini.indentscope").gen_animation.none(),
  },
})
```

I put this in my `init.lua` and worked. I tried to create a `lua/plugins/mini.indentscope.nvim` returning the configs (with `enabled=false` to completely disable the plugin) but didn't work.
